app/code/BVMyBlog/Blog/Ui/Component/Listing/Column/Thumbnail.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+namespace BVMyBlog\Blog\Ui\Component\Listing\Column;
+
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+
+/**
+ * Class Actions
+ *
+ * Thumbnail, image in grid
+ * @property \Magento\Framework\UrlInterface urlBuilder
+ */
+class Thumbnail extends \Magento\Ui\Component\Listing\Columns\Column
+{
+    const ALT_FIELD = 'name';
+    private $_objectManager = null;
+
+    /**
+     * @param ContextInterface $context
+     * @param UiComponentFactory $uiComponentFactory
+     * @param \Magento\Framework\UrlInterface $urlBuilder
+     * @param \Magento\Framework\ObjectManagerInterface $objectManager
+     * @param array $components
+     * @param array $data
+     */
+    public function __construct(
+        ContextInterface $context,
+        UiComponentFactory $uiComponentFactory,
+        \Magento\Framework\UrlInterface $urlBuilder,
+        \Magento\Framework\ObjectManagerInterface $objectManager,
+        array $components = [],
+        array $data = []
+    ) {
+        parent::__construct($context, $uiComponentFactory, $components, $data);
+        $this->urlBuilder = $urlBuilder;
```

Comment:
Please remove argument and field. Use ContextInterface::getUrl instead

Reply:
ContextInterface::getUrl doesn't include methods to return base url. Method getUrl return only params - it is not compatible with current usage.

Reply:
Really? Please try use ContextInterface::getUrl('', ['_direct' =>$filename ])